### PR TITLE
apps/*OverviewPages/templates: check for language_code and in case of…

### PR DESCRIPTION
**Pull request by khamui**
… de-ls hide intro image

### CODE REVIEW

File: apps/home/templates/apps_home/overview_page.html [hunk 1]
```diff
@@ -6,7 +6,8 @@
 <div class="fullwidth">
     <div id="layout-grid">
     <section id="layout-grid__area--herounit">
-        {% if page.intro_image %}
+        {{ request.LANGUAGE_CODE }}
+        {% if page.intro_image and request.LANGUAGE_CODE != 'de-ls' %}
```
Comment (philli-m) [inline apps/home/templates/apps_home/overview_page.html:10]: why use a variable and template tag here and just a template tag above, do I miss something?
also the language switch doesn't seem to work on overview anymore, does on gruenbuch though, maybe doing it other was is better?

**Review by philli-m — COMMENTED** [review]
When testing I had a thought, do we definitely not want this on the microsites overview too?

File: apps/home/templates/apps_home/overview_page.html (hunk 1 above)
Comment (fuzzylogic2000) [reply apps/home/templates/apps_home/overview_page.html:9]: @khamui What is the status here? Did you resolve it, can I have another look? And do you also want to add it to the microsites or did you already?

Comment (khamui) [reply apps/home/templates/apps_home/overview_page.html:9]: yes @fuzzylogic2000 it is resolved and works as expected.

**Review by fuzzylogic2000 — APPROVED** [review]
Works!

**Pull request merged**